### PR TITLE
[1/9] FastPfor codec: shared constants and helpers

### PR DESCRIPTION
**Pull request by DoFabien**
**Stacked PR**: 1/9 of the FastPFOR codec implementation.
> **Replaces**: #762 and #763 (split into smaller, reviewable PRs as requested).

## Overview

Adds shared constants and utility functions for the FastPFOR codec, used by both encoder and decoder.

## Added

- **Types**: `Int32Buf` and `Uint8Buf` type aliases for typed arrays
- **Constants**: `MASKS` (bit masks 0-32), `DEFAULT_PAGE_SIZE` (65536), `BLOCK_SIZE` (256)
- **Utilities**:
  - `greatestMultiple(value, factor)` - largest multiple of factor ≤ value
  - `roundUpToMultipleOf32(value)` - rounds up to next multiple of 32
  - `normalizePageSize(pageSize)` - validates and aligns page size to BLOCK_SIZE

## Testing

Unit tests covering all utilities with edge cases.

### CODE REVIEW

File: ts/src/decoding/fastPforShared.spec.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    describe("roundUpToMultipleOf32", () => {
+        it("rounds up to a multiple of 32", () => {
+            expect(roundUpToMultipleOf32(0)).toBe(0);
+            expect(roundUpToMultipleOf32(1)).toBe(32);
+            expect(roundUpToMultipleOf32(32)).toBe(32);
+            expect(roundUpToMultipleOf32(33)).toBe(64);
+        });
+    });
+
+    describe("MASKS", () => {
```
Comment (HarelM) [inline ts/src/decoding/fastPforShared.spec.ts]: [optinal] remove the test.

**Review by HarelM — COMMENTED** [review]
LGTM, thanks.

**Comment by DoFabien** [comment]
Done, test removed
Once this is merged, I'll open PR 2/9 with the byteIO utilities.

**Comment by HarelM** [comment]
Feel free to merge. Thanks!!

**Comment by DoFabien** [comment]
Thanks @HarelM! I don't have write access to the repo, could you please merge it for me when you have a moment?
I've already opened the next PR (#768) as a draft, which depends on this one. I will rebase it once this is merged.

**Comment by HarelM** [comment]
Ahh, I thought you are stacking the PRs on a branch in your repo. 
Do you think it would make more sense to do so? I.e. I'll have them reviewed the and then once all of them are ok we can merge together. 
What do you think?

**Comment by DoFabien** [comment]
I don’t have write access to create an upstream staging branch.
I can open the remaining parts as draft PRs to main for visibility, but diffs will be **cumulative** until earlier parts are merged.
If you want minimal per-PR diffs in the upstream UI, the alternative is an upstream staging branch (ex stack/fastpfor), and I’ll retarget the whole stack to it.

**Comment by HarelM** [comment]
No, I mean opening a series of PRs to a branch on your fork. 
When all PRs there are reviewed we can easily merge this branch on your fork containing all the merged PRs.
I'll do the review on your fork's PRs.

**Comment by DoFabien** [comment]
Thanks for the clarification ; sorry for the misunderstanding on my side.
I’ve now moved the whole stack to my fork and opened the PRs there against a staging branch, as you suggested.
You can review them here: https://github.com/DoFabien/maplibre-tile-spec/pulls

I closed the upstream PRs to avoid confusion.
Let me know if this setup works for you and I’ll follow up accordingly.
